From f81bba1fa22a1762d5c077f04835f593273fc4e3 Mon Sep 17 00:00:00 2001
From: Jim Gauld <james.gauld@windriver.com>
Date: Fri, 20 Mar 2020 19:23:49 -0400
Subject: [PATCH] Update Dex chart for Helm v3

---
 stable/dex/Chart.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/stable/dex/Chart.yaml b/stable/dex/Chart.yaml
index 1ca2f7b..794579b 100644
--- a/stable/dex/Chart.yaml
+++ b/stable/dex/Chart.yaml
@@ -1,3 +1,4 @@
+apiVersion: v1
 name: dex
 version: 0.8.0
 appVersion: 2.14.0
-- 
1.8.3.1

